packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/InteractionLayer.spec.js

```diff
@@ -156,7 +170,7 @@ describe('Component > InteractionLayer', function () {
           }
         }
         wrapper.find(StyledRect).simulate('pointerdown', fakeEvent)
-        wrapper.find(InteractionLayer).simulate('pointermove', fakeEvent)
+        wrapper.find('g').simulate('pointermove', fakeEvent)
```

Comment:
Could this select the root node of the wrapper, to avoid finding elements by name?